app/workers/education_form/create_spool_submissions_report.rb

```diff
@@ -39,11 +39,12 @@ def create_csv_array
     end
 
     def check_claim_for_stem(education_benefits_claim, parsed_form, stem_exists)
-      if stem_exists || (education_benefits_claim.form_type == '1995' && parsed_form['isEdithNourseRogersScholarship'])
-        return true
-      end
+      return true if stem_exists
+
+      # Remove when 1995s form type is in prod
```

Comment:
Is this working like a feature flag? Notes like this are pretty fragile... can you mark it with a TODO which will show up in some of our tools?

Reply:
Done!